ansible_rulebook/engine.py

```diff
@@ -362,6 +363,17 @@ async def _call_action(
         result = None
         if action in builtin_actions:
             try:
+                if action == "run_job_template":
+                    limit = dpath.get(
```

Comment:
action_args and job_args are typed as dicts, why do you use dpath?

Reply:
We don't have to use it, but dpath simplifies the code when I need to dig into nested dict. In this case I don't have to test whether action_args['job_args'] exists. dpath works on dicts https://pypi.org/project/dpath/